src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,24 @@
+@use '../utils/variables' as *;
+
+.stars {
+  display: flex;
+  align-items: center;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    padding-right: 4px;
+    background-size: 13px;
+    background-position: center;
+    background-repeat: no-repeat;
+    background-image: url(/src/images/star.svg);
```

Comment:
The path to the image should be relative to the CSS file, not the root of the project. Consider using '../images/star.svg' instead to ensure that the path resolves correctly when the CSS is built.